src/python/pants/backend/python/providers/python_build_standalone/constraints.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2024 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
This file could probably be moved to more library-like location in the source tree. For now, I'll just leave it here since this is the only user.